README.md

```diff
@@ -1,31 +1,33 @@
-# Civis Mistral Examples
+# Civis Workflows Examples
 
-This directory contains several examples of how to use Mistral with the
-Civis Platform to execute workflows with complex job dependencies.
+This directory contains several examples of how to use Workflows in Civis Platform
+to execute workflows with complex job dependencies.
 
-## Quckstart
+## Quickstart
 
 The optimal order to read through them is
 
 1. `quickstart.yml`
 2. `branching_and_dependencies.yml`
 3. `outputs_and_yaql.yml`
-4. `pause.yml`
-5. `yaml_variables.yml`
-6. `variable_blocks.yml`
-5. `sql_script_jobs.yml`
-6. `custom_email_notifications.yml`
-7. `with_items.yml`
-8. `with_items_dict.yml`
-9. `automatic_task_retry.yml`
-10. `task_defaults.yml`
-11. `export.yml` 
+4. `sql_scripts.yml`
+5. `script_parameters.yml`
+6. `pause.yml`
+7. `yaml_variables.yml`
+8. `variable_blocks.yml`
+9. `custom_email_notifications.yml`
+10. `with_items.yml`
+11. `with_items_dict.yml`
+12. `automatic_task_retry.yml`
+13. `task_defaults.yml`
+14. `export.yml`
```

Comment:
How do you feel about this ordering?  I moved `sql_scripts.yml` up because it's a pretty fundamental example, and I placed `script_parameters.yml` where it is because it uses SQL scripts and also introduces parameters, which are used in `yaml_variables.yml`.

Reply:
That makes sense to me